docs/usage.md

```diff
@@ -2393,6 +2405,12 @@ Where we have increased the default `4.GB` to `16.GB`. Make sure that you keep
 the `check_max` function, as this prevents your run asking for too much memory
 during retries.
 
+> Note that with this you will _not_ have the automatic retry mechanism. If
+> you want this, re-add the `check_max()` function on the `memory` line, and
+> add to the bottom of the entire file (outside the profiles block), the
+> block starting `def check_max(obj, type) {`, which is at the end of the
+> [nextflow.config file](https://github.com/nf-core/eager/blob/master/nextflow.config)
+
```

Comment:
What do you mean old version?

Reply:
To point to a version where you have this still in or to something similar.

Reply:
Ah, I see - good point. I don't think it'll be removed for a long time as it's in the nf-core template and AFAIK it would still be valid for DSL2. But I'll note that for the next patch (had to release today because I'm running a workshop tomorrow and needed to make sure it's working :sweat_smile: )